mlir/include/mlir/Dialect/Arith/Transforms/BufferDeallocationOpInterfaceImpl.h

```diff
@@ -0,0 +1,22 @@
+//===- BufferDeallocationOpInterfaceImpl.h --------------------------------===//
```

Comment:
The first line of header files should end with `--*- C++ -*-===//` if there's enough space.